.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: c
+dist: bionic
+
+compiler:
+  - clang
+  - gcc
+
+# Run only unit tests. Without OpenSSL, some additional tests in makefile do not work
+env:
+  - CONFIGURE=--with-kerberos5=auto                   TESTS="file-tests"
+  - CONFIGURE=--without-kerberos5                     TESTS="file-tests"
+  - CONFIGURE=--with-kerberos5=auto --without-openssl TESTS=""
+  - CONFIGURE=--without-kerberos5 --without-openssl   TESTS=""
+
+addons:
+  apt_packages:
+    - libssl-dev
+    - libkrb5-dev
+
+before_script:
+  - autoreconf
+  - ./configure $CONFIGURE
+
+script:
+  - make
+  - # make tests
+  - make unit $TESTS
```

Comment:
The travis is gone so it would probably make sense to drop this file and maybe replace it with some simple github action. They already have something in the https://github.com/openssh/openssh-portable/ mirror, which might be easy to reuse/adjust.